src/components/CreateAddress/PublicKeyImporter.jsx

```diff
@@ -218,14 +89,12 @@ class PublicKeyImporter extends React.Component {
   // Position
   //
 
-  moveUp = (event) => {
-    const { moveUp, number } = this.props;
+  const moveUpEvent = (event) => {
     event.preventDefault();
     moveUp(number);
   };
 
-  moveDown = (event) => {
-    const { moveDown, number } = this.props;
+  const moveDownEvent = (event) => {
```

Comment:
I think a better name would be:
```suggestion
  const handleMoveDown = (event) => {
```